lightning/src/onion_message/messenger.rs

```diff
@@ -300,6 +324,54 @@ where
 		}
 	}
 
+	fn respond_with_onion_message<T: CustomOnionMessageContents, EH: ResponseErrorHandler>(
+		&self, response: OnionMessageContents<T>, path_id: Option<[u8; 32]>,
+		reply_path: Option<BlindedPath>, error_handler: &EH
+	) {
+		let sender = match self.node_signer.get_node_id(Recipient::Node) {
```

Comment:
FWIW, we `unwrap` this elsewhere in the codebase

Reply:
Hmm... I was following the pattern in `send_onion_message` where there we don't `unwrap`.